App/src/main/java/cc/softwarefactory/lokki/android/MainActivity.java

```diff
@@ -458,6 +458,7 @@ public void onClick(DialogInterface dialog, int which) {
             try {
                 alertDialog.show();
             } catch (Exception ex) {
```

Comment:
The `alertDialog.show()` doesn't actually require a try-catch block so they can be removed.